controller/image.yaml

```diff
@@ -122,6 +122,7 @@ modules:
             version: "3.6"
           - name: jboss-kie-jdk-clean
             version: "11"
+          - name: jboss-kie-common
```

Comment:
is this module required?
IIRC this module only includes functions related to ocp k8s api.